maintenance_stock/data/demo_maintenance_stock.xml

```diff
@@ -0,0 +1,164 @@
+                'record': obj().env.ref('maintenance_stock.stock_warehouse_maintenance').cons_type_id,
+                'noupdate': True,
+            }, {
+                'xml_id': 'maintenance_stock.main_location_stock',
+                'record': obj().env.ref('maintenance_stock.stock_warehouse_maintenance').lot_stock_id,
+                'noupdate': True,
+            }, {
+                'xml_id': 'maintenance_stock.main_location_cons',
+                'record': obj().env.ref('maintenance_stock.stock_warehouse_maintenance').wh_cons_loc_id,
+                'noupdate': True,
+            }]"
+            />
+        </function>
+
+        <record id="product_toner" model="product.product">
+            <field name="default_code">TON_1234</field>
+            <field name="name">Toner</field>
+            <field name="type">product</field>
+            <field name="categ_id" ref="product.product_category_5" />
+            <field name="lst_price">100.0</field>
+            <field name="standard_price">70.0</field>
+            <field name="uom_id" ref="uom.product_uom_unit" />
+            <field name="uom_po_id" ref="uom.product_uom_unit" />
+            <field
+                name="image_1920"
+                type="base64"
+                file="maintenance_stock/static/img/toner.png"
+            />
+
+        </record>
+
+        <record id="stock_inventory_toner" model="stock.inventory">
+            <field name="name">Inventory for Toner</field>
+            <field
+                name="product_ids"
+                eval="[(6,0,[ref('maintenance_stock.product_toner'),])]"
+            />
+
+        </record>
+
+        <record id="stock_inventory_toner_line" model="stock.inventory.line">
+            <field name="product_id" ref="maintenance_stock.product_toner" />
+            <field name="product_uom_id" ref="uom.product_uom_unit" />
+            <field name="inventory_id" ref="maintenance_stock.stock_inventory_toner" />
+            <field name="product_qty">50.0</field>
+            <field name="location_id" ref="maintenance_stock.main_location_stock" />
+        </record>
+        <function name="action_start" model="stock.inventory">
+            <function
+                eval="[[('id', '=', ref('maintenance_stock.stock_inventory_toner'))]]"
+                model="stock.inventory"
+                name="search"
+            />
+        </function>
+        <function model="stock.inventory" name="action_validate">
+            <function
+                eval="[[('state','=','draft'),('id', '=', ref('maintenance_stock.stock_inventory_toner'))]]"
+                model="stock.inventory"
+                name="search"
+            />
+        </function>
+
+        <function model="stock.inventory" name="action_validate">
+            <function
+                eval="[[('state','in', ['draft, 'confirm']),('id', '=', ref('maintenance_stock.stock_inventory_toner'))]]"
```

Comment:
The single quote in `draft` is missing here, this causes an error when installing the demo data.

```suggestion
                eval="[[('state','in', ['draft', 'confirm']),('id', '=', ref('maintenance_stock.stock_inventory_toner'))]]"
```